notifications_utils/jinja_templates/email_template.jinja2

```diff
@@ -199,25 +194,23 @@
         <![endif]-->
                 <table role="presentation" width="100%" cellpadding="0" cellspacing="0" border="0" style="border-collapse: collapse">
                   <tr>
-                    {% if brand_colour %}
-                    <td height="10" width="1">{{ spacer(1, 10) }}</td>
-                    <td width="5">{{ spacer(1, 10) }}</td>
-                    {% endif %}
-                    <td height="10">{{ spacer(1, 10) }}</td>
-                    <td height="10">{{ spacer(1, 10) }}</td>
-                    <td height="10">{{ spacer(1, 10) }}</td>
+                    <td height="24" width="100%" colspan="{% if brand_colour %}3{% else %}2{% endif %}"><br /></td>
```

Comment:
`colspan` here should always be `2` – [this `<tr>`](https://github.com/alphagov/notifications-utils/pull/542/commits/d4a12691b3ad09c67c1db45a751e6680ff567da6#diff-7d3ef7fa9c747e8e757790bb586383e0R203) only ever has 2 columns.